cnv/cnv_release_notes/cnv-release-notes.adoc

```diff
@@ -92,6 +92,10 @@ interface of a Pod or virtual machine obtains a randomly generated MAC address r
 than a unique one from a pool. Although rare, randomly assigned MAC addresses can conflict.
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1816971[*BZ#1816971*])
 
+* Sometimes, when attempting to edit the subscription channel of the *{CNVProductNameStart} Operator* in the web console, clicking the
+*Channel* button of the *Subscription Overview* results in a JavaScript error.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1796410[*BZ#1796410*])
+
```

Comment:
in https://docs.openshift.com/container-platform/4.3/cnv/cnv_release_notes/cnv-release-notes.html#known-issues we also provided a workaround.
Can you please replicate it here simply replacing `CNV_CHANNEL=2.2` with `CNV_CHANNEL=2.3` ?

Reply:
@tiraboschi - is this the workaround you mean? (lines 99-106)

Reply:
yes, exactly

Reply:
LGTM